docs/OV_Runtime_UG/stateful_models_intro.md

```diff
@@ -0,0 +1,90 @@
+Stateful models and State API {#openvino_docs_OV_UG_stateful_models_intro}
+==============================
+
+@sphinxdirective
+
+.. toctree::
+    :maxdepth: 1
+    :hidden:
+
+    openvino_docs_OV_UG_ways_to_get_stateful_model
+
+@endsphinxdirective
+
+## What is a Stateful Network?
+
+ Several use cases require processing of data sequences. When length of a sequence is known and small enough, 
+ we can process it with RNN like networks that contain a cycle inside. But in some cases, like online speech recognition of time series 
```

Comment:
networks => model 
everywhere in the documents

Reply:
done